src/payments-processors/lnurl-payments-processor.ts

```diff
@@ -0,0 +1,69 @@
+import { AxiosInstance } from 'axios'
+import { Factory } from '../@types/base'
+
+import { CreateInvoiceRequest, GetInvoiceResponse, IPaymentsProcessor } from '../@types/clients'
+import { Invoice, InvoiceStatus, InvoiceUnit } from '../@types/invoice'
+import { createLogger } from '../factories/logger-factory'
+import { randomUUID } from 'crypto'
+import { Settings } from '../@types/settings'
+
+const debug = createLogger('alby-payments-processor')
```

Comment:
`alby`?

Reply:
We were first trying to bring support for [Alby](https://getalby.com/) but then realized we can generalize this for everyone implementing LNURL-verify, changing it to lnurl!

Reply:
lovely!